drivers/SmartThings/philips-hue/src/disco/light.lua

```diff
@@ -95,6 +91,7 @@ function M.handle_discovered_device(
         parent_assigned_child_key = parent_assigned_child_key
       }
 
+      log.debug(true, st_utils.stringify_table(st_metadata, "light create", true))
```

Comment:
Do we want this debug log?

Reply:
This specific one may have been removed but I think I'm going to leave most in during SVT testing.